src/styles/main.css

```
@@ -105,53 +174,120 @@ body {
+#app-title-h1 {
+  font-size: 2rem;
+  margin: 0 0;
+  padding: 0 0;
+  border: none;
+}
+
+#app-title-a {
+  color: #4c81af;
+  height: 100%;
+  text-decoration: none;
+  text-decoration: underline #739cc07b;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

text-decoration is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)